cache/remotecache/local/config.go

```diff
@@ -0,0 +1,67 @@
+package local
+
+import (
+	"strconv"
+
+	"github.com/moby/buildkit/util/compression"
+	digest "github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+)
+
+const (
+	attrDigest           = "digest"
+	attrSrc              = "src"
+	attrDest             = "dest"
+	attrOCIMediatypes    = "oci-mediatypes"
+	contentStoreIDPrefix = "local:"
+)
+
+type Config struct {
+	Digest         digest.Digest
+	Store          string
+	OCIMediatypes  bool
+	ContentStoreID string
+	Compression    compression.Config
+}
+
+func getConfig(attrs map[string]string, export bool) (*Config, error) {
```

Comment:
I'm not sure if this refactor makes the code easier to read - but I think it should be alright, I don't have strong opinions.